dap_taltech/utils/data_getters.py

```diff
@@ -0,0 +1,67 @@
+"""
+This script contains:
+    A class to load data from either a local directory or s3 data that is relevant
+    across tutorials.
+    The methods in the class have detailed docstrings that describe the data.
+"""
+import os
+import pandas as pd
+
+from dap_taltech import (PROJECT_DIR,
+                         PUBLIC_DATA_FOLDER_NAME,
+                         BUCKET_NAME,
+                         logger)
+
+class DataGetter(object):
+    """Class to load datasets relevant across different tutorials for TalTech HackWeek 2023.
+
+    Parameters
+    ----------
+    verbose : bool, optional
+        Set the logger level. If verbose, set logger level to INFO. Else, set logger level to ERROR.
+        Defaults to True.
+    local : bool, optional
+        If True, load data from local data directory. Else, download data from open dap-taltech s3 bucket.
+
+    """
+
+    def __init__(
+        self,
+        verbose=True,
+        local=True
+    ):
+        self.verbose = verbose,
```

Comment:
Doesn't the comma instantiate this is as a tuple?

Reply:
omg